src/js/apps/globals/nav_app.js

```diff
@@ -398,8 +410,14 @@ export default RouterApp.extend({
         patientModal.disableSubmit();
         patient.saveAll(patientClone.attributes)
           .then(({ data }) => {
-            Radio.trigger('event-router', 'patient:dashboard', data.id);
             patientModal.destroy();
+
+            if (patientFormId) {
+              Radio.trigger('event-router', 'form:patient', data.id, patientFormId);
```

Comment:
When we redirect to the patient form, this is what's shown for the breadcrumb link. A link to the patient dashboard. I think that makes sense...?

![Screenshot 2023-12-11 at 4 09 59 PM](https://github.com/RoundingWell/care-ops-frontend/assets/35355575/ea6ddfe6-3c18-4c66-babc-576654d99750)

Reply:
this is the desired behavior